src/docs/product/error-monitoring/issue-details/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Issue Details
+sidebar_order: 10
+description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
+---
+
+The Issue Details page helps you to gain further insight into the source of the issue. The information that is provided on the primary Issues page also displays on the Issue Details page. This page displays information about a specific error event that's part of an issue, however, you can navigate between the events of an issue using the "Older" and "Newer" buttons.
+
+TO DO Add screenshot here of Issue Details page
+
+Click on an issue on the main Issues page to open the Issue Details page.
+
+This page displays the Details tab when it first opens, but several other tabs are available:
+
+- Activity: A history of activity for the event where team members can share comments.
+- User Feedback: Any feedback collected from users through the <PlatformLink to="/enriching-events/user-feedback/">SDK's user feedback</PlatformLink>.
+- <PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>: Stored additional files, such as config or log files that are attached to the event.
+- Tags: More detailed information about the tags across all the events included in the issue.
+- Events: A list of all the events included in the issue.
+- Merged Issues: A list of issues that have been merged with this one.
+- Similar Issues: A list of issues that have a similar stack trace.
+
+In the right hand sidebar, [sentry.io](https://sentry.io) reflects how often the issue has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. You can also use this section to link to existing GitHub or Jira issues. Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the Global Header.
+
+In addition, this page provides several other key pieces of information.
+
+## Suspect Commits
+
+If you've configured [suspect commits](/product/releases/suspect-commits/), these are displayed. This allows you to immediately investigate commits that are likely to have generated the issue. The owners of suspect commits can be suggested assignees for an issue as well.
+
+## Tags
+
+Tags are the diagnostic information set by the SDK for the individual event. These are specific to each event in an issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.
```

Comment:
We should explain how this relates to the tags section in the sidebar.

Reply:
Edited